moveit_ros/planning_interface/test/python_move_group.py

```diff
@@ -56,12 +56,8 @@ def test_validation(self):
         error_code2, plan2, time = self.plan(current + 0.2)
 
         # both plans should have succeeded:
-        error_code = MoveItErrorCodes()
```

Comment:
This is how the API change will look like.
moveit_commander <del>is</del> should not be affected, but users of the raw planning interface have to adapt their code.